File: js/fizzbuzz.js
function fizzbuzz(range){
    var newstr = ''
    
    for (let i = 0; i < range; i++) {
        if (i % 5 === 0 || i % 3 === 0) {
            if (i % 5 === 0) {
                newstr += "Fizz"
            }
            if (i % 3 === 0) {
                newstr += "Buzz"
            }
            console.log(newstr)
            newstr = ''
        } else {
            console.log(i)
        }
    }
}


fizzbuzz(100)
// could definitely use some refining here.

File: py/text/fizzbuzz.py
for i in range(1, 101):
    new_str = ""
    if i % 3 == 0 or i % 5 == 0:
        if i % 3 == 0:
            new_str += 'Fizz'
        if i % 5 == 0:
            new_str += 'Buzz'
        print(new_str)
    else:
        print(i)
